flink-state-backends/flink-statebackend-rocksdb/src/test/java/org/apache/flink/contrib/streaming/state/RocksDBIncrementalCheckpointUtilsTest.java

```diff
@@ -95,39 +96,49 @@ public void testChooseTheBestStateHandleForInitial() {
         when(keyedStateHandle3.getKeyGroupRange()).thenReturn(new KeyGroupRange(8, 12));
         keyedStateHandles.add(keyedStateHandle3);
 
-        // this should choose no one handle.
-        Assert.assertNull(
+        Assert.assertEquals(
+                keyedStateHandle2,
```

Comment:
Why this test is different from previous one?
In my opion, we just make the option configurable, and it should not affect the testing result since we did not change the default overlap fraction threshold.